boards/px4/fmu-v2/default.cmake

```diff
@@ -11,6 +11,8 @@ px4_add_board(
 	IO px4_io-v2_default
 	#UAVCAN_INTERFACES 2
 	CONSTRAINED_FLASH
+	  NO_HELP
+	HELP_URL "https://docs.px4.io/master"
```

Comment:
Can you please change to: https://docs.px4.io/master/en

Reply:
Ah OK then The correct USL is `https://docs.px4.io/master/en/modules/modules_main.html` It has been updated.